Forta-Bots-Challenge-3/src/agent.ts

```diff
@@ -0,0 +1,57 @@
+import { BlockEvent, Finding, HandleBlock, getAlerts, getEthersProvider } from "forta-agent";
+import {
+  ETH_CHAIN_ID,
+  L1_ARBITRUM_ESCROW_ADDRESS,
+  L1_DAI_CONTRACT_ADDRESS,
+  L1_OPTIMISM_ESCROW_ADDRESS,
+  L2_DAI_ADDRESS,
+} from "./constants";
+import { providers } from "ethers";
+import { emitEscrowBalance, checkInvariant, contractAddresses, escrowBalances } from "./utils";
+
+let prevEscrowBalances: escrowBalances;
+
+export const provideHandleBlock = (
+  provider: providers.Provider,
+  contractAddresses: contractAddresses,
+  getAlerts: any
```

Comment:
Would advise against using the type of `any`. Could take a look [here](https://github.com/forta-network/forta-bot-sdk/blob/master/sdk/alerts.api.ts#L54C79-L54C79) for the type.